src/components/DateRangePicker.re

```diff
@@ -1,23 +1,15 @@
-type value = {
-  .
-  "initDate": Js.Date.t,
-  "endDate": Js.Date.t,
-};
-
-type state = {date: array(value)};
+type state = {date: array(Js.Date.t)};
```

Comment:
if this is just supposed to be a pair of dates (fixed size) prefer a tuple instead

Reply:
this code is outdated!
https://github.com/Astrocoders/astro-design/blob/feature/date-range-picker/src/components/DateRangePicker.re#L3